Quickstart documentation doesn't indicate that chapel-py is needed

**Describe the bug**
The quickstart instructions to get Arkouda set up is out of date - when I reached the `make` step, I received an error because I hadn't built `chapel-py`.  I think this means that the Chapel dependency instructions shouldn't just point to the quickstart page, they should indicate that `chapel-py` needs to be built.

**To Reproduce**
Steps to reproduce the behavior:
1. Follow the quickstart instructions from https://bears-r-us.github.io/arkouda/quickstart.html in a fresh environment.

**Expected behavior**
The Chapel dependencies section should mention installing chapel-py

**Error Message**
```
Traceback (most recent call last):
  File "<elided>/arkouda//src/registry/register_commands.py", line 6, in <module>
    import chapel
ModuleNotFoundError: No module named 'chapel'
Unable to register commands; make sure 'chapel-py' is installed
make: *** [register-commands] Error 1
```

**Is this a Blocking Issue**
- Are there any known work-arounds?
Installing chapel-py resolves the issue

**`ak.get_config()` Output**
Providing the output of `ak.get_config()` provides crucial debugging information.

N/A

**Additional context**
Add any other context about the problem here.

N/A

--- a/pydoc/quickstart.rst
+++ b/pydoc/quickstart.rst
@@ -11,7 +11,14 @@ Install Dependencies
 **********************
 
 1. Follow the `Chapel Quickstart Guide <https://chapel-lang.org/docs/usingchapel/QUICKSTART.html>`_
-2. Follow the `Anaconda Installation Guide <https://docs.anaconda.com/anaconda/install/index.html>`_
+2. Install chapel-py:
+
+   .. substitution-code-block:: bash
+
+       cd $CHPL_HOME
+       make chapel-py-venv
+
+3. Follow the `Anaconda Installation Guide <https://docs.anaconda.com/anaconda/install/index.html>`_
 
 **********************
 Install Arkouda

